lib/registries/addon/overview/index/template.hbs

```diff
@@ -3,9 +3,9 @@
         {{t 'registries.overview.component_of'}}
         <OsfLink
             @route='resolve-guid'
-            @models={{array this.root.id}}
+            @models={{array this.rootId}}
```

Comment:
Please see the comment above in lib/registries/addon/overview/index/controller.ts regarding this code change.

Reply:
This has been reverted in favor of changes in the model.